package.json

```diff
@@ -72,6 +74,9 @@
     "release-it": "13.7.1",
     "typescript": "^3.6.4"
   },
+  "peerDependencies": {
+    "babel-plugin-macros": "^3.1.0"
+  },
```

Comment:
I'm a bit worried about this, since `babel-plugin-macro` is not strictly a dependency, but an optional one. An idea would be to use [peerDependenciesMeta](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta) in order to specify they're optional, even if this would require additional checking in actual code. Can you take a look at this?

Reply:
Sounds reasonable.